.github/workflows/performance_test.yaml

```diff
@@ -0,0 +1,30 @@
+name: Performance testing
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+      - name: Run integration tests
+        env:
+          PYTHONPROFILEIMPORTTIME: "1"
+        run:
+          time snow --help
```

Comment:
I think this is more of a check than test, we don't compare with anything right? Maybe we should make it a normal pytest test? We can run snowcli via `subprocess` and measure the time. This gives us more elasticity in saving the results as well as comparing them